aimeedyki/hello-Books
# #152379230 deploy app to heroku

#### What does this PR do?
Sets up application to build and run successfully on Heroku.

#### Description of Task to be completed?
- set up different Heroku environments for production
- add config variables for hosted app
- successfully deploy base app to heroku and run it

#### How should this be manually tested?
Open the app url for [hello-books](https://hello-books-amarachi.herokuapp.com) 
You should then see the live app running.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[deploy app to heroku](https://www.pivotaltracker.com/story/show/152379230)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2017-10-31 at 7 39 47 pm" src="https://user-images.githubusercontent.com/28511567/32242514-4f5027a0-be73-11e7-9f33-365eb7678484.png">
